docs/x509.rst

```diff
@@ -77,6 +100,36 @@ Loading Certificates
     >>> cert.serial
     2
 
+Loading Certificate Signing Requests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. function:: load_pem_x509_csr(data, backend)
+
+    .. versionadded:: 0.9
+
+    Deserialize a certificate signing request (CSR) from PEM encoded data. PEM
+    requests are base64 decoded and have delimiters that look like
+    ``-----BEGIN CERTIFICATE REQUEST-----``. This is also known as PKCS#10
+    format.
```

Comment:
I think this sentence reads a bit better as "This format is also known as PKCS#10"